tests/helpers.py

```diff
@@ -110,14 +110,17 @@ def create_secret(interface_type):
         secret_data = defaults.CSI_RBD_SECRET.copy()
         del secret_data['data']['kubernetes']
         secret_data['data']['admin'] = get_admin_key()
+        component = constants.RBD_COMPONENT
     elif interface_type == constants.CEPHFILESYSTEM:
         secret_data = defaults.CSI_CEPHFS_SECRET.copy()
         del secret_data['data']['userID']
         del secret_data['data']['userKey']
         secret_data['data']['adminID'] = constants.ADMIN_BASE64
         secret_data['data']['adminKey'] = get_admin_key()
+        component = constants.CEPHFS_COMPONENT
+
     secret_data['metadata']['name'] = create_unique_resource_name(
-        'test', 'secret'
+        f'test-{component}', 'secret'
```

Comment:
I think 'component' should replace the second arg, in this case - 'secret'

Reply:
secret would be the type of resource.
resource name would look like tthis --> secret-test-cephfs-1713121946